src/card.css

```diff
@@ -0,0 +1,126 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.container__card {
+  margin: auto;
+}
+
+.card__link {
+  display: block;
+  outline-color: var(--secondary-color);
+}
+
+.card__img {
+  display: block;
+  max-width: 100%;
+  padding: 0 3px;
+  margin-bottom: 40px;
+  text-align: center;
+  outline-color: var(--secondary-color);
+}
+
+.card__title {
+  display: block;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  outline-color: var(--secondary-color);
+
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 140%;
```

Comment:
it's OK to use `%` whet it's something like 100%, 150%, etc.
but 140% seem to be weird